SimulationCore/Simulation/CoreColony.cs

```diff
@@ -73,32 +73,32 @@ internal sealed class CoreColony
         /// <summary>
         /// Die Drehgeschwindigkeiten aller Kasten des Volkes in Grad pro Runde.
         /// </summary>
-        internal readonly int[] Drehgeschwindigkeit;
+        internal readonly int[] RotationSpeed;
 
         /// <summary>
         /// Die Lebenspunkte aller Kasten des Volkes.
         /// </summary>
-        internal readonly int[] Energie;
+        internal readonly int[] EnergyI;
 
         /// <summary>
         /// Die Geschwindigkeiten aller Kasten des Volkes in der internen Einheit.
         /// </summary>
-        internal readonly int[] GeschwindigkeitI;
+        internal readonly int[] SpeedI;
 
         /// <summary>
         /// Die maximalen Lastwerte aller Kasten des Volkes.
         /// </summary>
-        internal readonly int[] Last;
+        internal readonly int[] Burden;
```

Comment:
We usually translate "Last" with "Load" all over the place. To be consistent I would suggest to use "Load" also here.